components/map-preview.js

```diff
@@ -5,7 +5,12 @@ import { useEffect, useState } from "react";
 import { useRouter } from "next/router";
 import path from "path";
 
-const Route = ({ track }) => {
+// Generate random hex color
+const randomColorGenerator = () => {
+  return "#" + Math.floor(Math.random() * 16777215).toString(16);
+};
```

Comment:
Use a nicer color palette with something like https://gka.github.io/chroma.js/

Reply:
Done in [6631b1a](https://github.com/hutec/ber-cph/pull/3/commits/6631b1a99d21f11b27124fb55d7603ab30a2ba8d)